### PR TITLE
layout-register

### PR DESCRIPTION
**Pull request by ghost**
## Description
layout register
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

### CODE REVIEW

File: resources/views/user/register/register.blade.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                        <input type="text" class="form-control" id="InputAddress" name="address" placeholder="{{__('user.admin.create.enter_address')}}">
+                        </div>
+                        <div class="form-group">
+                        <label for="InputPhone">{{ __('user.admin.create.phone') }}</label>
+                        <input type="text" class="form-control" id="InputPhone" name="phone" placeholder="{{__('user.admin.create.enter_phone')}}">
+                        </div>
+                        <div class="form-group">
+                        <label for="InputAvatar">{{ __('user.admin.create.avatar') }}</label>
+                        <input type="file" class="form-control" id="InputAvatar" name="avatar" placeholder="{{__('user.admin.create.enter_avatar')}}">
+                        </div>
```
Comment (at-thaitran) [inline resources/views/user/register/register.blade.php]: thẻ label và input lùi tab vô

File: resources/views/user/register/register.blade.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                        <input type="file" class="form-control" id="InputAvatar" name="avatar" placeholder="{{__('user.admin.create.enter_avatar')}}">
+                        </div>           
+                    </div>
+                    <div class="box-footer">
+                        <button type="submit" class="btn btn-primary" name="submit">{{ __('user.admin.create.submit') }}</button>
+                    </div>
+                </form>
+           </div>
+      </div>
+    </div>
```
Comment (at-thaitran) [inline resources/views/user/register/register.blade.php]: tab 2 space

**Comment by at-thaitran** [comment]
LGTM

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**